[libc][docs] some code-blocks in the docs are missing highlighting languages

On 1460b4964c7ada2f7536006722c8585b5bd0a1b5, I am getting the following error:

```
Warning, treated as error:
.../llvm-project/libc/docs/dev/config_options.rst:14:Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

   {
     "printf": {
       "LIBC_CONF_PRINTF_DISABLE_FLOAT": {
         "value": false,
         "doc": "Disable printing floating point values in printf and friends."
       },
       ...
     }
   }
ninja: build stopped: subcommand failed.

```

The code-blocks in this doc are old, so I might've broken something on my local machine causing this, but these are the only code blocks with no highlighting language.

--- a/libc/docs/dev/config_options.rst
+++ b/libc/docs/dev/config_options.rst
@@ -121,7 +121,7 @@ should convert the CMake config options to appropriate compiler and/or linker
 flags. Those compile/link flags can be used in listing the affected targets as
 follows:
 
-.. code-block:: none
+.. code-block:: cmake
 
    add_object_library(
     ...

--- a/libc/docs/dev/config_options.rst
+++ b/libc/docs/dev/config_options.rst
@@ -11,15 +11,14 @@ hierarchical JSON files. At the top of the hierarchy is a JSON file by name
 options which affect all platforms. The default value for the option and a short
 description about it listed against each option. For example:
 
-.. code-block::
+.. code-block:: json
 
    {
      "printf": {
        "LIBC_CONF_PRINTF_DISABLE_FLOAT": {
          "value": false,
          "doc": "Disable printing floating point values in printf and friends."
-       },
-       ...
+       }
      }
    }
 
@@ -28,7 +27,7 @@ has a value of ``false``. A platform, say the baremetal platform, can choose
 to override this value in its ``config.json`` file in the ``config/baremetal``
 directory with the following contents:
 
-.. code-block::
+.. code-block:: json
 
    {
      "printf": {
@@ -61,14 +60,13 @@ The value corresponding to each grouping tag is also a dictionary called the
 options belonging to that grouping tag. For the ``printf`` tag in the above
 example, the option-dictionary is:
 
-.. code-block::
+.. code-block:: json
 
    {
      "LIBC_CONF_PRINTF_DISABLE_FLOAT": {
        "value": false,
        "doc":
-     },
-     ...
+     }
    }
 
 The value corresponding to an option key in the option-dictionary is another
@@ -86,7 +84,7 @@ Option name format
 
 The option names, or the keys of a option-dictionary, have the following format:
 
-.. code-block::
+.. code-block:: none
 
    LIBC_CONF_<UPPER_CASE_TAG_NAME>_<ACTION_INDICATING_THE_INTENDED_SEMANTICS>
 
@@ -123,7 +121,7 @@ should convert the CMake config options to appropriate compiler and/or linker
 flags. Those compile/link flags can be used in listing the affected targets as
 follows:
 
-.. code-block::
+.. code-block:: none
 
    add_object_library(
     ...

